reference/6/Microsoft.PowerShell.Utility/Import-Csv.md

```diff
@@ -72,58 +79,54 @@ WUDFHost               0  834     2199310204928 51945472  87441408  24984  C:\Wi
 
-Also, because there is no entry type in the formatting files for the CSV version of the process objects, these objects are not formatted in the same way that standard process objects are formatted.
-
-To display the objects, use the formatting cmdlets, such as Format-Table and Format-List, or pipe the objects to Out-GridView.
+Because there is no entry type in the formatting files for the CSV version of the process objects, these objects are not formatted in the same way that standard process objects are formatted.
+To display the objects, use the formatting cmdlets, such as `Format-Table` and `Format-List`.
 
 ### Example 2: Specify the delimiter
-```
-PS C:\> Get-Process | Export-Csv Processes.csv -Delimiter :
-PS C:\> $P = Import-Csv Processes.csv -Delimiter :
+```powershell
+Get-Process | Export-Csv -Path .\Processes.csv -Delimiter :
+$P = Import-Csv -Path .\Processes.csv -Delimiter :
 ```
 
-This example shows how to use the *Delimiter* parameter of the **Import-Csv** cmdlet.
+This example shows how to use the **Delimiter** parameter of the `Import-Csv` cmdlet.
 
 In this example, the processes are exported to a file that uses a colon (:) as a delimiter.
 
-When importing, the **Import-Csv** file uses the *Delimiter* parameter to indicate the delimiter that is used in the file.
+When importing, the `Import-Csv` file uses the **Delimiter** parameter to indicate the delimiter that is used in the file.
 
 ### Example 3: Specify the current culture for the delimiter
+```powershell
+Get-Process | Export-Csv -Path .\Processes.csv -UseCulture
+$P = Import-Csv -Path .\Processes.csv -UseCulture
+(Get-Culture).TextInfo.ListSeparator
 ```
-PS C:\> Get-Process | Export-Csv Processes.csv -UseCulture
-PS C:\> $P = Import-Csv Processes.csv -UseCulture
-PS C:\> (Get-Culture).TextInfo.ListSeparator
-,
-```
-
-This example shows how to use the *UseCulture* parameter of the **Import-Csv** cmdlet.
 
-In this example the processes are exported to a file that uses the culture as a delimiter. The next command imports the objects in the Processes.csv file into the $P variable. It uses the *UseCulture* parameter to direct **Import-Csv** to use the list separator defined for the current culture.
+This example shows how to use the **UseCulture** parameter of the `Import-Csv` cmdlet.
 
-The second command displays the list separator for the current culture. It uses the Get-Culture cmdlet to get the current culture. It uses the dot (.) method to get the TextInfo property of the current culture and the ListSeparator property of the object in TextInfo.
+In this example the processes are exported to a file that uses the culture as a delimiter. The next command imports the objects in the Processes.csv file into the `$P` variable. The **UseCulture** parameter directs `Import-Csv` to use the list separator defined for the current culture.
```

Comment:
Change to "uses the default list separator of the current culture setting."

Reply:
Changed the sentence and committed other changes for Example 3.